Earthfile

```diff
@@ -267,6 +267,15 @@ base-image:
           systemctl enable cos-setup-network.service
     END
 
+    # TEST KAIROS-AGENT FROM BRANCH
+    ARG KAIROS_AGENT_DEV
+    ARG KAIROS_AGENT_DEV_BRANCH=master
+    IF [ "$KAIROS_AGENT_DEV" = "true" ]
+        RUN rm -rf /usr/bin/kairos-agent
```

Comment:
shouldn't this be part of the framework instead?

Reply:
Was gonna add it there, but this is mainly for dev... and Im guessing for building full isos to try them manually with changes. Not sure if needed on the framework? Im worried that it migth end up in a pushed image or something :/